### PR TITLE
updated tools to add Workbox

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

File: src/content/en/tools/_index.yaml [hunk 1]
```diff
@@ -39,14 +39,16 @@ landing_page:
...
   - heading: Libraries
     items:
-    - heading: Service Worker Libraries
+    - heading: Workbox
       description: >
-        Our Service Worker Libraries are provided to help simplify your
-        development experience, that free your app from the network, without
-        having to write any boilerplate service worker code!
+        Workbox is a set of tools and libraries to help you simplify your
+        Progressive Web App development experience and take advantage of the
```
Comment (jpmedley) [inline src/content/en/tools/_index.yaml]: Delete 'experience'. I don't think it adds anything.

File: src/content/en/tools/_index.yaml [hunk 2]
```diff
@@ -39,14 +39,16 @@ landing_page:
...
+        many powerful features of Service Worker. It helps you free your app
```
Comment (jpmedley) [inline src/content/en/tools/_index.yaml]: 'Service Worker` to `service workers`

File: src/content/en/tools/_index.yaml [hunk 3]
```diff
@@ -39,14 +39,16 @@ landing_page:
...
+        from the network, without having to write any boilerplate service worker
+        code!
```
Comment (jpmedley) [inline src/content/en/tools/_index.yaml]: Period instead of exclamation point.

File: src/content/en/tools/workbox/index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+easy to take advantage of powerful service worker features, eliminate boilerplate code, and automate service worker generation.
+
+<figure class="attempt-right">
+  <img src="/web/tools/workbox/thumb.png" alt="Workbo logo">
+</figure>
+
+Workbox is a collection of loosely-coupled libraries and tools, that focus on
+different service worker features and use-cases. The two core tools that Workbox provides are:
+
+**workbox-sw&mdash;**A service worker library to make managing fetch requests
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: A list of definitions are one place it's permissible to use a sentence fragment. Please do so throughout this list. Use parallel sentence constructions as much as possible and start definitions with present tense verbs. 

* A service worker library that makes fetch requests and caching as easy as possible.
*A command-line tool that generates...

File: src/content/en/tools/workbox/index.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+and caching as easy as possible.
+
+**workbox-cli&mdash;**A command-line tool to generate a service worker and a
+file manifest, that makes use of the workbox-sw module to become a fully-functioning service worker.
+
+These two high-level tools are built up from a number of lower-level modules
+that can be used independently or mixed-and-matched:
+
+**workbox-build&mdash;**A module that gan be used to generate a file manifest or
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: For the rest of these you can leave out module and library.
* Generates a file manifest...
* Implements various ...

File: src/content/en/tools/workbox/index.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+**workbox-google-analytics&mdash;**A service worker helper library to retry
+offline Google Analytics requests when a connection is available.
+
+**workbox-background-sync&mdash;**A generic helper library that queues failed
+requests and uses the Background Sync API to replay those requests when the user
+comes back online.
+
+Workbox is the next generation of our service worker libraries and tools,
+previously called `sw-toolbox` and `sw-precache`. If you were familiar with
+those tools, Workbox will be very familiar as well. Workbox is the result of
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: 'Workbox will be very familiar as well' to 'Workbox will feel familiar'

(My suggestion removes unneeded words substitutes a stronger verb. I don't care what verb is used as long as 'will be' goes away.)

File: src/content/en/tools/workbox/index.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+redesigning these tools to be more modular, flexible, and extensible.
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: I'm worried that this paragraph makes Wrokbox sound like Service Worker Libraries 2.0. It isn't literally that. @addyosmani

File: src/content/en/tools/workbox/index.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+The [Workbox command-line tool](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-cli)
+(`workbox-cli`) is a standalone CLI to generate a service worker and a file
+manifest, that uses the `workbox-sw` module to end up with a fully-functioning
+service worker.
+
+Don't want to use yet another CLI? Workbox is also compatible with many
+different build tools. The
+[`workbox-build`](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-build)
+module is a portable Node module that can generate a file manifest or service
+worker to work with `workbox-sw`. The `workbox-build` module can be used with
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: 'The `workbox-build` module can be used' to 'Use the `workbox-build` module'

File: src/content/en/tools/workbox/index.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+Analytics requests. For example, you might set a custom dimension to indicate
+that a request was replayed. |
+| Modified Hit Parameters | Lets you programmatically modify a hit's parameters
+to, for example, track the elapsed time between when a hit is attempted and when
+ it is replayed. |
+
+## Background Sync
+
+[Workbox Background Sync](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-background-sync)
+will queue failed requests and use the
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: Use present tense verbs here to be consistent with the rest of your descriptions.

File: src/content/en/tools/workbox/index.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+[Background Sync API](/web/updates/2015/12/background-sync) to replay those
+requests when the user comes back online.
+
+### Features
+
+| Feature | Summary |
+|---------|---------|
+| Background Queuing | Manages a queue for replaying requests that have failed.
+|
+| Granular Queue Control | Specify how long a request should live in the queue
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: 'Specify' to 'Specifies'

File: src/content/en/tools/workbox/index.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+before it is abandoned, and what to do when it succeeds. |
+| Broadcast Channel Management | Supports updating the queued request's status
+over a Broadcast Channel, so that you do things like provide the user with a
```
Comment (jpmedley) [inline src/content/en/tools/workbox/index.md]: 'so that you do things' to 'so that you can do things'

**Comment by tjsavage** [comment]
Thank you @jpmedley!! Make edits as suggested.

**Comment by petele** [comment]
Is there a specific time this should go live?

**Comment by addyosmani** [comment]
It would be great if this were live before the mobile web keynote on Wednesday (today). If it slips before I/O, we'd love to have it live before Jeff's talk on Thursday. Thanks, @petele!

**Pull request merged**